### ISSUE TITLE
🐛 Vuejs domains rules don't activate even under "vue": "all"

### ISSUE THREAD

**Issue by destroyer22719** [biomejs/biome#8681]
### Environment information

Vue domain rules must be manually activated undern ursery, even under domain the configuration `"vue": "all"` doesn't activate any of them.

To test this, go to this [repo](https://github.com/destroyer22719/biome-vue-test), this vue code should give an error:

```vue
<script setup lang="ts">
import { ref } from 'vue';

const myBoolean = ref(true);
</script>

<template>
<div v-if="myBoolean" v-for="x in [1,2,3]"></div>
</template>

<style scoped>
</style>
```

Because it violates the rule 

But running `pnpm lint` says it's okay.

To fix it we must manually add:
```json
"nursery": {
	"noVueVIfWithVFor": "error"
}
```

But it should have been activated by the domain setting under 

```json
"domains": {
	"vue": "all"
}
```

### What happened?

Go to this repo:
https://github.com/destroyer22719/biome-vue-test

and follow the above instructions

### Expected result

all rules, even under nursery should be activated

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
All rules are still nursery, they require explicit opt-in 

See the docs https://biomejs.dev/linter/domains/#vue-activation

EDIT

I believe the docs are incorrect, the `all` shouldn't enable nursery rules.

**Comment by destroyer22719** [comment]
Yeah, maybe fix the wording of the doc?
>Since all rules in this domain are nursery rules, no rules will be activated when using the “recommended” setting. Use “all” to enable all nursery rules in this domain.

**Comment by ematipico** [comment]
Yeah I'll do that ASAP
